### PR TITLE
Add the Symfony3 base image

### PR DESCRIPTION
**Pull request by sroze**

### CODE REVIEW

File: symfony/3-apache/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM quay.io/continuouspipe/php-apache:7.0_v1
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -yq install \
+    acl \
```
Comment (sroze) [inline symfony/3-apache/Dockerfile]: Is it to have `setfacl` ? If yes, wouldn't in makes sense to move it to the Ubuntu base image actually, as we'll have this problem often I believe!

File: symfony/3-apache/usr/local/share/symfony/install.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+mkdir -p /app/var
+
+set +e
+is_nfs
+IS_NFS=$?
+set -e
+
+if [ "$IS_NFS" -ne 0 ]; then
+  # Ensure code is owned by a user other than the web server user
+  chown -R build:build /app
```
Comment (sroze) [inline symfony/3-apache/usr/local/share/symfony/install.sh]: That's quite aggresive

File: symfony/3-apache/usr/local/share/symfony/install.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  chmod -R go-rw /app/var
+  # Fix permissions so the web server user can write to /app/var for symfony cache files
+  setfacl -R -m u:www-data:rwX -m u:build:rwX /app/var
```
Comment (sroze) [inline symfony/3-apache/usr/local/share/symfony/install.sh]: `/etc/exports`:
```
/Users/sroze 192.168.99.100 -alldirs -mapall=502:20
```

File: symfony/3-apache/usr/local/share/symfony/install.sh (hunk 2 above)
Comment (sroze) [inline symfony/3-apache/usr/local/share/symfony/install.sh]: But fair probably.

Comment (kierenevans) [inline symfony/3-apache/usr/local/share/symfony/install.sh]: Same as deploy user owned code in traditional deployments.

File: symfony/3-apache/Dockerfile (hunk 1 above)
Comment (kierenevans) [inline symfony/3-apache/Dockerfile]: Will move out :)

**Comment by sroze** [comment]
🎉

**Pull request merged**